storage-client/src/bin/driver.rs

```diff
@@ -1,7 +1,8 @@
-use arrow_array::Float64Array;
+use arrow::array::Float64Array;
```

Comment:
What's the difference?

Reply:
I believe in arrow `50.0.0` we should use `arrow::array` instead of `arrow_array`